feat(RUN-971): Add page listing Execution errors (#2845)
dfinity/portal: docs/references/execution-errors.mdx

import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";

> # Execution Errors

Reviewer: ```suggestion
# Execution errors
```

<MarkdownChipRow labels={["Reference"]} />

## Overview
A list of possible errors returned when executing canisters.

## Errors

> ### Method Not found

Reviewer: ```suggestion
### Method not found
```

A canister was called with a method name not exported by that canister.

An example of this error is:
```
  Error from Canister xxx-xxx: Canister has no update method 'foobar'.
```

> To fix this error, consider

Reviewer: ```suggestion
To fix this error, consider:
```

> 1. Checking that the method name exacly matches the name exported by the callee.
> 1. Checking that the method type (update, query, etc...) is the same as the type of the exported method.
> 1. The canister code of the callee has not been modified to a version that depricates the method.

Reviewer: ```suggestion
- Checking that the method name exactly matches the name exported by the callee.
- Checking that the method type (update, query, etc...) is the same as the type of the exported method.
- The canister code of the callee has not been modified to a version that depreciates the method.
```

Author: I fixed "exactly" and the bullets, but I think "depricates" is actually correct.

Reviewer: "depricates" is not a dictionary recognized word: https://www.merriam-webster.com/dictionary/depricates

Reviewer: However depreciate is also incorrect; it should be 'deprecate' if you mean the method is no longer supported.

Author: I believe "depricates" is in Merriam Webster, it's just a conjugation of "depricate": https://www.merriam-webster.com/dictionary/deprecate

Reviewer: Yes, it should be `deprecate`, with an `e` not an `i`.

Author: oh, yeah of course.
